dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo

```diff
@@ -25,8 +25,8 @@ import Modelica;
 
```

Comment:
Not in you PR but since I see it, let's correct it here : No line between model and import + two spaces before all imports

Reply:
Also import Connectors is useless

Reply:
Also two spaces before annotation